code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -1,6 +1,7 @@
 /obj/structure/closet/cardboard
 	name = "large cardboard box"
 	desc = "Just a box..."
+	icon = 'icons/obj/boxes.dmi'
```

Comment:
I can't actually comment directly on the icon file, but it's named `Boxes.dmi`, and it's generally preferred to not have filenames capitalized like that.